plugin/src/main/resources/plugin.yml

```diff
@@ -8,6 +8,9 @@ api-version: 1.16
 depend:
   - ProtocolLib
   - StructureBlockLib
+libraries:
+  - com.github.shynixn.structureblocklib:structureblocklib-bukkit-api:2.2.2
+  - com.github.shynixn.structureblocklib:structureblocklib-bukkit-core:2.2.2
```

Comment:
Whats this for?

Reply:
Required according to the [StructureBlockLib GitHub page](https://github.com/Shynixn/StructureBlockLib#for-version--117).